packages/source-fetcher/lib/etherscan.ts

```diff
@@ -62,7 +62,7 @@ const EtherscanFetcher: FetcherConstructor = class EtherscanFetcher
     if (networkName === undefined || !supportedNetworks.includes(networkName)) {
       throw new InvalidNetworkError(networkId, "etherscan");
     }
-    this.suffix = networkName === "" ? "mainnet": `${networkName}`;
+    this.networkName = "" ? "mainnet": networkName;
```

Comment:
```suggestion
    this.networkName = networkName;
```

Reply:
Awesome thanks for the suggestions. Made this change!